### ISSUE TITLE
Add "array" type

### ISSUE THREAD

**Issue by justinwinslow** [activeprospect/leadconduit-types#36]
we ran into an issue where a user was presented with a description that made it unclear the field was an array. The confusion was compounded by the fact they were shown all of the available operators.

**Comment by justinwinslow** [comment]
Ran into another issue where the Suppression List Found In field is an array about the UI didn't pare back the operators and there was no indication the value would be an array and the user selected `is equal to` resulting in none of the rules with the Found In field validating as expected.

**Comment by alexkwolfe** [comment]
Isn't the SL response variable an array type?

**Comment by justinwinslow** [comment]
Yes, but there's nothing to inform the UI what operators are applicable.

**Comment by justinwinslow** [comment]
We could add something manually if you don't think an array type belongs in the types api.
